uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Validator/Date.php

```diff
@@ -22,7 +24,10 @@ class CRM_HRLeaveAndAbsences_Validator_Date {
   public static function isValid($date, $format = 'Y-m-d')
   {
     $dateTime = DateTime::createFromFormat($format, $date);
-
+    if (!$dateTime && $format != 'YmdHis') {
+      $format = 'YmdHis';
+      $dateTime = DateTime::createFromFormat($format, $date);
+    }
```

Comment:
What if, instead of this new if, we change the default value for format to 'YmdHis'? I think that the dates will already be in this format most of the time when we get to the validation